src/client/eppo-client.ts

```diff
@@ -350,6 +350,7 @@ export default class EppoClient {
       this.banditVariationConfigurationStore ?? null,
       this.banditModelConfigurationStore ?? null,
     );
+    this.configurationRequestor = configurationRequestor;
```

Comment:
`getConfiguration():` has only worked because it creates a StoreBackConfig using the same store ConfigRequestor is backed by 😢 🐼